# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/compiler/translator/Initialize.cpp
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/compiler/translator/Initialize.cpp
--- a/gfx/angle/checkout/src/compiler/translator/Initialize.cpp	Fri Nov 08 16:41:42 2019 +0000
+++ b/gfx/angle/checkout/src/compiler/translator/Initialize.cpp	Fri Nov 08 17:06:00 2019 +0000
@@ -61,6 +61,10 @@
     {
         extBehavior[TExtension::ARM_shader_framebuffer_fetch] = EBhUndefined;
     }
+    if (resources.OVR_multiview)
+    {
+        extBehavior[TExtension::OVR_multiview] = EBhUndefined;
+    }
     if (resources.OVR_multiview2)
     {
         extBehavior[TExtension::OVR_multiview2] = EBhUndefined;
@@ -77,6 +81,10 @@
     {
         extBehavior[TExtension::OES_texture_storage_multisample_2d_array] = EBhUndefined;
     }
+    if (resources.OES_texture_3D)
+    {
+        extBehavior[TExtension::OES_texture_3D] = EBhUndefined;
+    }
     if (resources.ANGLE_texture_multisample)
     {
         extBehavior[TExtension::ANGLE_texture_multisample] = EBhUndefined;
@@ -85,6 +93,10 @@
     {
         extBehavior[TExtension::ANGLE_multi_draw] = EBhUndefined;
     }
+    if (resources.ANGLE_base_vertex_base_instance)
+    {
+        extBehavior[TExtension::ANGLE_base_vertex_base_instance] = EBhUndefined;
+    }
 }
 
 void ResetExtensionBehavior(TExtensionBehavior &extBehavior)